secboot/secboot.go

```diff
@@ -29,6 +29,7 @@ import (
 
 	"github.com/snapcore/snapd/asserts"
 	"github.com/snapcore/snapd/bootloader"
+	"github.com/snapcore/snapd/overlord/devicestate/fde"
```

Comment:
this import is a bit odd,  maybe is a matter that devicestate is not ideal for fde, not sure.  Anyway I think for now I would prefer the approach where FDEHasReavelKey is injected in

Reply:
I pushed a commit that uses a indirect init for this, please let me know if this is what you had in mind:
https://github.com/snapcore/snapd/pull/9718/commits/64b96e129751e4c1bf6527ad8da46baaa16145c2